### ISSUE TITLE
[Issue]: Marking an item with progress as watched does not update LastPlayedDate

### ISSUE THREAD

**Issue by Gylesie** [jellyfin/jellyfin#8492]
### Please describe your bug

When marking an item which has progress associated with it as watched _without providing `datePlayed` query parameter_, Jellyfin does not update `LastPlayedDate` saved in Jellyfin's database.

1. Watch an item and stop the playback arbitrarily in such a way that it is not automatically marked as watched by Jellyfin
2. Wait few minutes
3. [Send a Jellyfin API request](https://api.jellyfin.org/#tag/Playstate/operation/MarkPlayedItem) to mark the item as watched without the `datePlayed` parameter
4. Check `LastPlayedDate`
5. The time does not correspond to the time at which the request in step 3 was sent.

On the other hand, it seems that marking an item in the same way which does not have progress updates the `LastPlayedDate`.

Relevant issue: https://github.com/jellyfin/jellyfin-plugin-trakt/issues/168

### Jellyfin Version

Other

### if other:

10.8.5

### Environment

```markdown
- OS: Debian
- Virtualization: Docker
- Clients: API
- Browser: -
- FFmpeg Version: 5.1.1-Jellyfin
- Playback Method: -
- Hardware Acceleration: -
- Plugins: Trakt
- Reverse Proxy: nginx
- Base URL: none
- Networking: Bridge
- Storage: local
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Gylesie** [comment]
I have traced it back to this line: https://github.com/jellyfin/jellyfin/blob/719e5eae16a3488de449a5b2a7c56132c8f1e5c1/MediaBrowser.Controller/Entities/BaseItem.cs#L1854

It seems that it was intended that way to work. Can I call for a discussion about this behaviour? From my perspective and use-case it would make sense to not fallback to media's lastPlayedDate at all when marking an item as watched, but rather fallback to current time only.

**Comment by cvium** [comment]
> I have traced it back to this line: https://github.com/jellyfin/jellyfin/blob/719e5eae16a3488de449a5b2a7c56132c8f1e5c1/MediaBrowser.Controller/Entities/BaseItem.cs#L1854
> 
> 
> 
> It seems that it was intended that way to work. Can I call for a discussion about this behaviour? From my perspective and use-case it would make sense to not fallback to media's lastPlayedDate at all when marking an item as watched, but rather fallback to current time only.

The decision to make it behave this way predates the Jellyfin fork. I agree with you to an extent. If a video contains 20+ minutes of rolling credits, it may not be marked as played. Marking it at a later point, it would make sense to keep lastplayeddate. I don't think there's one specific fallback that fits all.

**Comment by Gylesie** [comment]
That's interesting. I am actually writing it because of the same problem of long rolling credits. I have encouraged a problem where in the end the progress was overwritten because of `lastPlayedDate` when importing from Trakt (the linked issue). We could argue that it should be fixed on Trakt plugin's side of things, but _I_ agree with @Shadowghost on this one because of the following reasons:

1. **Consistency**
Currently, when a user is using a native Jellyfin client (mainly speaking about WebUI here) the request that is being sent to the server includes `datePlayed` which is the current time. There is no other way in the UI to mark an item as watched without the native client implicitly sending the `datePlayed` parameter. 

2. **Intuition** (subjective)
Now I can speak only from my experience. Wherever I clicked the mark as watched button, I always subconsciously thought that the item was being marked _(as of right now)_. This was maybe because of other services that behaved like this by default. Only the help of logs and Trakt plugin contributor helped me discover that it actually behaves like it does right now.

However I also definitely see a use case where `lastPlayedDate` user would want it to stay the same as you suggested. Based on my opinion, I would keep the current time (`DateTime.UtcNow`) as the only fallback and I would add to the native WebUI the capability for the user to change the `datePlayed` parameter which the client sends to the server.

**Comment by Gylesie** [comment]
> Marking it at a later point, it would make sense to keep lastplayeddate

I have just noticed that `LastPlayedDate` does not equal to the date at which the playback was stopped. It equals to the date at which the last playback _started_. I. e. starting playback at 16:00, watching it half-way through, stopping the playback at 16:30 and then marking it as watched would result with a media that has `LastPlayedDate` equal to 16:00.

**Comment by cvium** [comment]
Fixed by #8523
